packages/devtools_app/lib/src/analytics/analytics.dart

```diff
@@ -245,27 +235,17 @@ class GtagExceptionDevTools extends GtagException {
   external String get flutter_client_id;
 }
 
-ValueNotifier<bool> _gaEnabledNotifier = ValueNotifier(false);
-
-ValueListenable<bool> get gaEnabledNotifier => _gaEnabledNotifier;
-
-// Exposed function to JS via allowInterop.
-bool gaEnabled() => _gaEnabledNotifier.value;
-
 /// Request DevTools property value 'enabled' (GA enabled) stored in the file
 /// '~/.flutter-devtools/.devtools'.
 Future<bool> isAnalyticsEnabled() async {
-  _gaEnabledNotifier.value = await server.isAnalyticsEnabled();
-  return _gaEnabledNotifier.value;
+  print('in analytics isAnalyticsEnabled');
```

Comment:
remove all print statements before commit or change to logging.

Reply:
done.